readthedocs/projects/migrations/0082_init_generic_webhooks.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.2.24 on 2021-09-22 21:02
+
+from django.db import migrations, models
+import django.utils.timezone
+import django_extensions.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0081_add_another_header'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='WebHookEvent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(choices=[('build:triggered', 'Build triggered'), ('build:passed', 'Build passed'), ('build:failed', 'Build failed')], max_length=256, unique=True)),
```

Comment:
I think it will be good to auto-create these three in a data migration instead manually on production and locally.